pages/index.js

```diff
@@ -53,15 +55,18 @@ export function getServerSideProps(context) {
   }
 
   return {
-    props: {}
+    props: {},
+    redirect: {
+      destination: config.WIX_URL,
+    },
   }
 
 }
 
 export default function BannerPage({ }) {
   return <BaseContainer>
     <Helmet>
-      <title>Better Social</title>
+      <title>Helio</title>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the title tag content from "Better Social" to "Helio" is a simple text modification for rebranding purposes. However, it's crucial to ensure that all other references to the old brand name are also updated throughout the application to maintain consistency.
```diff
- <title>Better Social</title>
+ <title>Helio</title>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->